### ISSUE TITLE
v2.2.5 (windows): 0h/s in ethash dual mining

### ISSUE THREAD

**Issue by UselessGuru** [doktor83/SRBMiner-Multi#268]
Soon after the DAG has been created ethash rate drops to 0.

`SRBMinerMulti-v2.2.5\SRBMiner-MULTI.exe --esm 2 --disable-cpu --disable-gpu-amd --disable-gpu-intel --algorithm ethash --pool ethash.eu.mine.zergpool.com:9999 --wallet ###################### --password c=BTC,mc=EGEM,ID=#########,pl=0 --tls false --algorithm kaspa --pool kheavyhash.eu.mine.zergpool.com:5555 --wallet ###################### --password c=BTC,mc=KAS,ID=#########,pl=0 --tls false --gpu-id 0 --disable-cpu --api-rig-name ######### --api-enable --api-port 4006`

```
...
[2023-05-09 07:42:59] GPU0[t0] result accepted [   48ms] [kaspa][1]
[2023-05-09 07:43:01] Job received [90] [kaspa][1]
[2023-05-09 07:43:01] ================================================================
[2023-05-09 07:43:01] [0]    [ethash]
[2023-05-09 07:43:01] GPU0 : 28.33 MH/s         [       0|    0|    0|   349.76 kH/W]
[2023-05-09 07:43:01] Total: 28.33 MH/s
[2023-05-09 07:43:01] ================================================================
[2023-05-09 07:43:01] [1]    [kaspa]
[2023-05-09 07:43:01] GPU0 : 112.50 MH/s        [       2|    0|    0|  1388.89 kH/W]
[2023-05-09 07:43:01] Total: 112.50 MH/s
[2023-05-09 07:43:01] ================================================================
[2023-05-09 07:43:01] GPU0 : [T:  46c CC:  1455MHz MC:   7420MHz FAN:    43 P:   81W]
[2023-05-09 07:43:01] Total:                                                     81W
[2023-05-09 07:43:01] ================================================================
[2023-05-09 07:43:05] ================================================================
[2023-05-09 07:43:05] [0]    [ethash]
[2023-05-09 07:43:05] GPU0 : 28.33 MH/s         [       0|    0|    0|   833.22 kH/W]
[2023-05-09 07:43:05] Total: 28.33 MH/s
[2023-05-09 07:43:05] ================================================================
[2023-05-09 07:43:05] [1]    [kaspa]
[2023-05-09 07:43:05] GPU0 : 112.35 MH/s        [       2|    0|    0|  3304.42 kH/W]
[2023-05-09 07:43:05] Total: 112.35 MH/s
[2023-05-09 07:43:05] ================================================================
[2023-05-09 07:43:05] GPU0 : [T:  43c CC:  2025MHz MC:   7420MHz FAN:    43 P:   34W]
[2023-05-09 07:43:05] Total:                                                     34W
[2023-05-09 07:43:05] ================================================================
[2023-05-09 07:43:08] ================================================================
**[2023-05-09 07:43:08] [0]    [ethash]
[2023-05-09 07:43:08] GPU0 : 0.00 H/s           [       0|    0|    0|             -]
[2023-05-09 07:43:08] Total: 0.00 H/s**
[2023-05-09 07:43:08] ================================================================
[2023-05-09 07:43:08] [1]    [kaspa]
[2023-05-09 07:43:08] GPU0 : 112.35 MH/s        [       2|    0|    0|  9362.53 kH/W]
[2023-05-09 07:43:08] Total: 112.35 MH/s
[2023-05-09 07:43:08] ================================================================
[2023-05-09 07:43:08] GPU0 : [T:  39c CC:   405MHz MC:    810MHz FAN:    40 P:   12W]
[2023-05-09 07:43:08] Total:                                                     12W
[2023-05-09 07:43:08] ================================================================
[2023-05-09 07:43:11] Job received [111] [kaspa][1]
[2023-05-09 07:43:21] Job received [131] [kaspa][1]
[2023-05-09 07:43:26] ================================================================
[2023-05-09 07:43:26] [0]    [ethash]
[2023-05-09 07:43:26] GPU0 : 0.00 H/s           [       0|    0|    0|             -]
[2023-05-09 07:43:26] Total: 0.00 H/s
[2023-05-09 07:43:26] ================================================================
[2023-05-09 07:43:26] [1]    [kaspa]
[2023-05-09 07:43:26] GPU0 : 112.35 MH/s        [       2|    0|    0|  2675.01 kH/W]
[2023-05-09 07:43:26] Total: 112.35 MH/s
[2023-05-09 07:43:26] ================================================================
[2023-05-09 07:43:26] GPU0 : [T:  42c CC:  2025MHz MC:   7420MHz FAN:    42 P:   42W]
[2023-05-09 07:43:26] Total:                                                     42W
[2023-05-09 07:43:26] ================================================================
```

This happens with
- Ethash & heavyhash
- Ethash & kaspa
- Ethash & sha256dt
- Ethash & sha512_256d_radiant

**Comment by doktor83** [comment]
cause your gpu crashed you can see from the power consumption

**Comment by doktor83** [comment]
I like it when 'all other miners work fine no issues and never crash'. 
It's really helpful in finding out what your issue is. 
Cause it's something at your system because if this was an issue everyone has im sure i would already know about it.

For start try running ethash in single mode and see what happens.

**Comment by UselessGuru** [comment]
> I like it when 'all other miners work fine no issues and never crash'.
> It's really helpful in finding out what your issue is.

I write this to make clear that there is not a general driver or HW issue with my computer.
It is specific to SRB miner.

> For start try running ethash in single mode and see what happens.
Working just fine. Only dual modes are affected.

**Comment by doktor83** [comment]
Fixed in v227 ?

**Comment by UselessGuru** [comment]
Fixed in v2.2.7

Thank you!
